src/tiled/command.cpp

```diff
@@ -30,39 +30,70 @@
 using namespace Tiled;
 using namespace Tiled::Internal;
 
+QString Command::finalWorkingDirectory() const
+{
+    QString finalWorkingDirectory = workingDirectory;
+
+    finalWorkingDirectory = replaceVariables(finalWorkingDirectory, false);
+
+    QString finalExecutable = replaceVariables(executable);
+    QFileInfo mFile(finalExecutable);   //Check if executable is a path or not
```

Comment:
I still don't really understand why you need this check. What would go wrong, if you left it out and just always replaced `%executablepath` with `mFile.absolutePath()`? And in what way is it more useful for the user to replace it with an empty string instead?